src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1843,13 +1844,14 @@ PeerImp::getLedger (std::shared_ptr<protocol::TMGetLedger> const& m)
     if (packet.itype () == protocol::liTS_CANDIDATE)
     {
         // Request is for a transaction candidate set
-        p_journal_.trace <<
+        if (p_journal_.trace) p_journal_.trace <<
             "GetLedger: Tx candidate set";
```

Comment:
Does this check really buy us anything for trivial messages (ie. raw strings that don't do any evaluation)?